### PR TITLE
Added homework 04

### PR DESCRIPTION
**Pull request by marti456**

### CODE REVIEW

File: homework_04/14_martin_damyanov/homework.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def get_x_coord(self):
+        """returns x coordinate"""
+        return self.x_coord
+
+    def get_y_coord(self):
+        """returns y coordinate"""
+        return self.y_coord
+
+class Circle:
+    """Implenets class for circle"""
```
Comment (Vladikolev0321) [inline homework_04/14_martin_damyanov/homework.py:36]: You have a typo should be "Implement".

File: homework_04/14_martin_damyanov/tests.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+def test_create_invalid_circle():
+    """Tests wrong creating of circle"""
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        Circle(5, 6)
+    assert pytest_wrapped_e.type == SystemExit
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        point = Point(2.5, 9.7)
+        Circle(point, 'radius')
+    assert pytest_wrapped_e.type == SystemExit
+
```
Comment (Vladikolev0321) [inline homework_04/14_martin_damyanov/tests.py]: Tests are ok.

File: .github/workflows/homework_04_14_martin_damyanov.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+
+    - name: Run the tests
+      run: |
+        cd homework_04
+        cd 14_martin_damyanov
+        pytest
+
+    - name: Check the code with pylint
```
Comment (Vladikolev0321) [inline .github/workflows/homework_04_14_martin_damyanov.yaml:38]: Everything seems fine.

**Review by Vladikolev0321 — APPROVED** [review]
Nice work.

File: homework_04/14_martin_damyanov/homework.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            return RelativePosition.INTERSECTING
+        raise ValueError('A very specific case happened')
+
+    def get_center(self):
+        """retuns center point"""
+        return self.center
+
+    def get_radius(self):
+        """retuns radius"""
+        return self.radius
```
Comment (Vicktoria853) [inline homework_04/14_martin_damyanov/homework.py:70]: Tests are 9.33
You can disable C1001 (old style class) and R0903 (too few public methods). (#pylint: disable=R0903). 
Anything else is good.

File: homework_04/14_martin_damyanov/tests.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    """Tests relative position of intersecting"""
+    circle1 = Circle(Point(2.0, 1.0), 5.0)
+    circle2 = Circle(Point(6.0, 4.0), 6.0)
+    assert circle1.find_relative_position(circle2) == RelativePosition.INTERSECTING
+
+def test_same():
+    """Tests if two circles are the same"""
+    circle1 = Circle(Point(2.0, 2.0), 5.0)
+    circle2 = Circle(Point(2.0, 2.0), 5.0)
+    assert circle1.find_relative_position(circle2) == RelativePosition.SAME
```
Comment (Vicktoria853) [inline homework_04/14_martin_damyanov/tests.py:53]: Results from pylint  are 10
Great job!

**Review by Vicktoria853 — APPROVED** [review]

**Pull request merged**